R/CageData_Integration.R

```diff
@@ -157,18 +156,19 @@ findMaxPeaks = function(cageOverlaps, filteredrawCageData){
 #' @param fiveUTRs The 5' leader sequences as GRangesList
 #' @param cageData The location of the cage-file
 #' @param extension The number of basses upstream to add on transcripts
-findNewTSS = function(fiveUTRs, cageData, extension){
+findNewTSS <- function(fiveUTRs, cageData, extension){
 
   shiftedfiveUTRs <- extendsTSSexons(fiveUTRs, extension)
   cageOverlaps <- findOverlaps(query = cageData, subject = shiftedfiveUTRs)
   maxPeakPosition <- findMaxPeaks(cageOverlaps, cageData)
   return(maxPeakPosition)
 }
 
-#' After all transcript start sites have been updated from cage, grangeslist back together
+#' After all transcript start sites have been updated from cage,
+#'  put grangeslist back together
 #' @param firstExons The first exon of every transcript from 5' leaders
 #' @param fiveUTRs The 5' leader sequences as GRangesList
-makeGrlAndFilter = function(firstExons, fiveUTRs){
+makeGrlAndFilter <- function(firstExons, fiveUTRs){
```

Comment:
I guess its one of your helper functions, it breaks the rule - one function one purpose as it does GRL and Filters.